packages/auth/package.json

```diff
@@ -44,6 +44,6 @@
     "jest-openapi": "^0.14.2",
     "nock": "^13.2.4",
     "openapi-types": "^12.0.0",
-    "typescript": "^4.2.4"
+    "typescript": "^4.3.0"
```

Comment:
I'm still getting the warning and so is GitHub Actions
https://github.com/interledger/rafiki/pull/387#discussion_r937048994
https://github.com/interledger/rafiki/runs/7682195245?check_suite_focus=true

I wonder if typescript should be updated in the root `package.json`

Reply:
I'll try it and see what happens

Reply:
Bumping typescript breaks a lot of stuff

Reply:
That doesn't make any sense because `^4.2.4` actually installs `4.7.2` currently as the caret means any non breaking changes.